docs/user_guide/upgrading.rst

```diff
@@ -0,0 +1,38 @@
+Upgrading
+==========
+See also `CHANGES.md` <https://github.com/fairlearn/fairlearn/blob/master/CHANGES.md>`_.
```

Comment:
Minor update: should be
```
`CHANGES.md <link>`_
```
that is no backquote after the intended representation.

Reply:
You can check what it will render as by clicking through the Circle CI check at the bottom of the PR. This is the resulting page: https://493-133444044-gh.circle-artifacts.com/0/docs/_build/html/user_guide/upgrading.html

(which I should document somewhere!)

Reply:
thanks! 👍